README.md

```diff
@@ -177,6 +177,13 @@ Installing the dependencies
 conda install -c conda-forge -c bioconda --file requirements.txt
 ```
 
+## Citation
+If you use GoldRush in your research, please cite:
+
+Wong, Johnathan, Lauren Coombe, Vladimir Nikolić, Emily Zhang, Ka Ming Nip, Puneet Sidhu, René L Warren, and Inanç Birol. 2022. ‘GoldRush: A *de novo* Long Read Genome Assembler with Linear Time Complexity’. BioRxiv, January, 2022.10.25.513734. https://doi.org/10.1101/2022.10.25.513734.

```

Comment:
```suggestion
Johnathan Wong, Lauren Coombe, Vladimir Nikolić, Emily Zhang, Ka Ming Nip, Puneet Sidhu, René L Warren, and Inanç Birol. 2022. ‘GoldRush: A *de novo* Long Read Genome Assembler with Linear Time Complexity’. BioRxiv, January, 2022.10.25.513734. https://doi.org/10.1101/2022.10.25.513734.

```
What's the "January" from?

Reply:
I'll need to check the citation format. I just chose one that didn't put et al.